tables/he-IL-comp8.utb

```diff
@@ -1,18 +1,18 @@
-# liblouis: Multi-language computer braille table for Hebrew, Arabic and English (Israel).
+# liblouis: computer braille table for Hebrew (Israel).
 #
 #-index-name: Hebrew, Israel, computer
-#-display-name: Israeli computer braille
-#
-#+locale: mul-IL
+#-display-name: Hebrew computer braille
+#+locale: he
```

Comment:
Can this table be used for Hebrew all over the world or is it specific for Israel?

generate-display-names should be adapted too.

Reply:
If the table is not specific to Israel anymore (we don't really have the answer for this yet I think), the metadata should be as follows:

    #-index-name: Hebrew, computer
    #-display-name: Hebrew computer braille

and the line in display-names should be:

    * ../../tables/he-IL-comp8.utb      Hebrew, computer       Hebrew computer braille

Note the `*` at the beginning which indicates that the last column is automatically generated. If the `*` is not there and the last column matches the `#-display-name` field in the table the tool will complaint too.

Reply:
OK, thanks, it seems to work now